### ISSUE TITLE
Aesthetics based on shape

### ISSUE THREAD

**Issue by dcjones** [dcjones/Gadfly.jl#66]
In particular for the point geometry. See Wilkinson p. 281 which cites several others.

**Comment by scidom** [comment]
I agree, circles, elllipses, squares are pretty common to have. R's `pch` palette may also be useful as a guide.

**Comment by joshjob42** [comment]
I'd like to bring some more attention to this one. I'm not adept enough at Compose.jl, etc. to really do this. I can try to work on it to some extent.

It's just that for some stupid reason editors insist on figures being distinguishable in black and white. Which is monstrously stupid in my opinion (goodbye a whole dimension of visual information) but it's the case. I'd very much like to not have to resort to recreating all my plots in MATLAB if I could help it.

**Comment by tbreloff** [comment]
See: https://github.com/tbreloff/Plots.jl/issues/12

**Comment by tbreloff** [comment]
Also see: https://github.com/tbreloff/Plots.jl/issues/45

**Comment by bjarthur** [comment]
should we close this now that we have https://github.com/GiovineItalia/Gadfly.jl/blob/master/src/shapes.jl ?  only thing missing in my mind is a way to have them be opened or closed.
